client/app/scripts/superdesk-desks/desks.js

```diff
@@ -871,6 +885,9 @@
                                 _.extend(origDesk, scope.desk.edit);
                             }
 
+                            scope.desks._items = _.sortBy(scope.desks._items, function(desk) {
+                                return desk.name.toLowerCase();
+                            });
```

Comment:
Similar suggestion here, the same comparison function is repeated multiple times.